pkgs/tools/security/eid-mw/default.nix

```diff
@@ -27,30 +40,30 @@ stdenv.mkDerivation rec {
     export SSL_PREFIX=$(realpath openssl)
     substituteInPlace plugins_tools/eid-viewer/Makefile.in \
       --replace "c_rehash" "openssl rehash"
-    '';
+  '';
+  # pinentry uses hardcoded `/usr/bin/pinentry`, so use the built-in (uglier) dialogs for pinentry.
```

Comment:
Why are we not patching this path?

Reply:
It didn't seem worth it to introduce a custom patch for something that is not actually required.

Reply:
You can probably replace it with substituteInPlace to get the not ugly dialog.

Reply:
As mentioned in chat, I don't really know what pinentry flavor to select. The gnupg pinentry flavor only selects it for gpg-agent by adding an argument to an exec line in a systemd service. It might make sense to introduce a nixos module that selects a pinentry flavor and links it to `/run/current-system/sw/bin/pinentry`, but that seems out-of-scope for this PR and also doesn't fix anything for non-NixOS users.